wpilib/wpilib/drive/differentialdrive.py

```diff
@@ -112,6 +112,8 @@ def __init__(self, leftMotor: SpeedController, rightMotor: SpeedController) -> N
         """
         super().__init__()
 
+        assert leftMotor is not None, "Left Motor should not be None"
```

Comment:
I think David actually added this check to exluded_commits so I'll get rid of these

Reply:
I put then back in.